stumpy/floss.py

```diff
@@ -570,8 +570,8 @@ def update(self, t):
             T_subseq_isconstant = core.rolling_isconstant(self._T, self._m)
             M_T, Σ_T = core.compute_mean_std(self._T, self._m)
             D = core.mass(
-                self._finite_Q,
-                self._finite_T,
+                self._T[-self._m :],
+                self._T,
                 M_T,
```

Comment:
I revise the code to avoid the postprocessing after the `if/else` block. However, if you think we should keep the postprocessing to increase the safety, I can undo the changes.

Reply:
I kind of like how explicit the original version was. I say we revert back to the original